Sources/AuthFoundation/Network/APIResponse.swift

```diff
@@ -14,6 +14,8 @@ import Foundation
 
 /// Describes a response from an Okta request, which includes the supplied result, and other associated response metadata.
 public struct APIResponse<T: Decodable>: Decodable {
+    @available(*, deprecated, renamed: "APIRateLimit")
+    public typealias RateLimit = APIRateLimit
     /// Links between response resources.
```

Comment:
Can you ensure there's an empty line between this?

Reply:
Good callout. I've updated this.